### PR TITLE
add FAQ questions + events tab

### PR DESCRIPTION
**Pull request by rymaju**

### CODE REVIEW

**Comment by rymaju** [comment]
Disregard the title of the PR, this is an update to add two commonly asked questions to the FAQ and also add the events tab, linking to a google doc with a list of all upcoming events.

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "react-redux": "^7.2.0",
     "react-reveal": "^1.2.2",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1",
+    "react-scripts": "^3.4.2",
```
Comment (jackblanc) [inline package.json:25]: Why the change?

File: src/components/ApplyBrandDesigner.tsx [hunk 2]
```diff
@@ -143,6 +143,16 @@ const ApplyBrandDesigner: React.FC = () => {
               </Typography>
             </Box>
           </Fade>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="body1" gutterBottom className={classes.text}>
+                <i>
+                  All product teams will be working and meeting remotely. <br />
```
Comment (jackblanc) [inline src/components/ApplyBrandDesigner.tsx:150]: Not necessarily 100% true but close enough. We might do in-person team events for fun if permissible by NEU.

File: src/components/ApplyDeveloper.tsx [hunk 3]
```diff
@@ -145,6 +145,16 @@ const ApplyDeveloper: React.FC = () => {
               </Typography>
             </Box>
           </Fade>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="body1" gutterBottom className={classes.text}>
+                <i>
+                  All product teams will be working and meeting remotely. <br />
```
Comment (jackblanc) [inline src/components/ApplyDeveloper.tsx:152]: Make this consistent with the other pages, if you change any of them.

Also, `<br />`? Is there a better way?

File: src/components/FAQ.tsx [hunk 4]
```diff
@@ -13,6 +13,16 @@ Every page must have a Helmet tag for SEO purposes.
 */
 
 const faq = [
+  {
+    title: 'Will C4C events be held virtually?',
+    body:
+      'Yes, all C4C events will be held virtually and recorded. Our event planners are working hard to ensure that all our recordings will be available and easily accessible.',
```
Comment (jackblanc) [inline src/components/FAQ.tsx:19]: this is true

File: src/components/FAQ.tsx [hunk 5]
```diff
@@ -13,6 +13,16 @@ Every page must have a Helmet tag for SEO purposes.
...
+  },
+  {
+    title: "Can I still apply to join C4C if I'm not on campus this semester?",
+    body:
+      'Absolutely! Our product teams are now working fully remotely, so no matter where you are you can still join C4C and help the Boston community. Our teams are comfortable working with others in different timezones, and we will never consider your current location to be a negative factor in our application process.',
```
Comment (jackblanc) [inline src/components/FAQ.tsx:24]: okay i see what you mean, nobody _has_ to be in Boston. But we will try and do in person if it's possible + safe

File: src/components/NavBar.tsx [hunk 6]
```diff
@@ -75,6 +75,9 @@ const NavBar: React.FC = () => {
     setAnchorEl(null);
   };
 
+  const eventsDocsLink =
```
Comment (jackblanc) [inline src/components/NavBar.tsx:78]: Wait, I don't know how I feel about this part, it seems weird for a nav link to go external. I think it's worth our effort to add a calendar to the site tbh

**Review by jackblanc — APPROVED** [review]
You _can_ merge if you'd like. But, I'm not a huge fan of linking to a google doc from the navbar. I'd like to add a page, ideally. 

Everything else looks great

File: package.json (hunk 1 above)
Comment (rymaju) [reply package.json:25]: hmm I think I npm audit'ed because of a high risk warning and this just updated too

File: src/components/ApplyBrandDesigner.tsx (hunk 2 above)
Comment (rymaju) [reply src/components/ApplyBrandDesigner.tsx:150]: Sure, let me know if you want me to rephrase it

File: src/components/FAQ.tsx (hunk 5 above)
Comment (rymaju) [reply src/components/FAQ.tsx:24]: I see, I can rewrite the copy a bit to reflect that

File: src/components/NavBar.tsx (hunk 6 above)
Comment (rymaju) [reply src/components/NavBar.tsx:78]: Good point, my concern was having 2 sources of information for the events. I was thinking that it would be really easy to desync the actual event schedule and whatever we have on the website. It would be best if we could somehow read the google doc and then display that on the page. Im sure theres some npm module for that or worst case we can simply scrape it.
